app/models/certificate.rb

```diff
@@ -19,4 +19,12 @@ def signing?
   def x509
     to_x509(value)
   end
+
+  def certificate_issuer_common_name
```

Comment:
This is much better, just the name of the methods; I think
certificate_issuer_common_name => issuer_common_name
and we don't really need the helper method certificate_issuer_find
```
  def issuer_common_name
    x509.subject.to_a.find { |issuer, _, _| issuer == 'CN' }[1]
  end
```
would be fine